.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,34 @@
+# This package will publish the package on NPM
+name: Publish Package
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest    
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install
+      - run: yarn run eslint
```

Comment:
Is there any possibility to reuse an existing workflow ?

Reply:
No idea!